packages/wrangler/src/user/user.ts

```diff
@@ -241,7 +239,7 @@ import {
 	getRevokeUrlFromEnv,
 	getTokenUrlFromEnv,
 } from "./auth-variables";
-import { ChooseAccount, getAccountChoices } from "./choose-account";
+import { type ChooseAccountItem, getAccountChoices } from "./choose-account";
```

Comment:
To maintain consistency we've been using 
`import type { ChooseAccountItem } from "./choose-account";`